jgrapht-core/src/main/java/org/jgrapht/experimental/alg/color/BrownBacktrackColoring.java

```diff
@@ -28,12 +28,15 @@
  * @param <V> the graph vertex type
  * @param <E> the graph edge type
  * 
- * @author micha
+ * @author Michael Behrisch
  */
 public class BrownBacktrackColoring<V, E>
```

Comment:
I'm late to the party, but yeah, the experimental package is vestigial from the time when we didn't have a good code review system available, so it would be good to phase it out.